blockchain/types/transaction_signing.go

```diff
@@ -59,7 +59,7 @@ func MakeSigner(config *params.ChainConfig, blockNumber *big.Int) Signer {
 	var signer Signer
 
 	if config.IsEthTxTypeForkEnabled(blockNumber) {
-		signer = NewLondonSigner(config.ChainID)
+		signer = NewPragueSigner(config.ChainID)
```

Comment:
I think the logic should be:
```
	if config.IsPragueForkEnabled(blockNumber) {
		signer = NewPragueSigner(config.ChainID)
	} else if config.IsEthTxTypeForkEnabled(blockNumber) {
		signer = NewLondonSigner(config.ChainID)
	} else {
		signer = NewEIP155Signer(config.ChainID)
	}
```

Reply:
When adding IsPragueForkEnabled, we must specify PragueCompatibleBlock.
https://github.com/kaiachain/kaia/blob/f329a688e17b7741d35f2b4736c05d3d7a77d430/params/config.go#L54

Has the next hard fork block been decided? If not, would you like to temporarily enter the maximum value of big.Int?

Reply:
Oh actually we have the field in [dev branch](https://github.com/kaiachain/kaia/blob/dev/params/config.go#L62).
You can add the field and assign "nil", which indicates that the fork block is undetermined (similar to max value of big.Int).

Reply:
Thank you for letting me know. 
I would implement it so that conflict resolution is easy.